easybuild/easyconfigs/s/SWIG/SWIG-3.0.12-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,21 @@
+name = 'SWIG'
+version = '3.0.12'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
```

Comment:
@RvDijk should just be `toolchainopts = {'pic': True}`? (same in other `SWIG` easyconfig)

Reply:
@boegel I got these from the `foss 2016b` or comparable version.. Will remove!